Simulator/Vertices/NG911/All911Vertices.h

```diff
@@ -81,6 +81,8 @@ class All911Vertices : public AllVertices {
    vector<vector<uint64_t>> logAnswerTime_;
    /// The end time for every call in the log
    vector<vector<uint64_t>> logEndTime_;
+   /// Logs if the call was abandoned
+   vector<vector<bool>> logWasAbandoned_;
```

Comment:
Are these data members meant to hold a log of something? Right now, starting their name with `log`, they sound like verbs -- like they produce a log. If they're meant to be a log, then it might be better to end with `log`, like `answerTimeLog_`. OTOH, `log` overlaps with our logging framework. Are they meant to store a history? Then it would be consistent with the nomenclature in our recorders to name them like `answerTimeHistory_`.

Reply:
My thought was that these represent the Call Log, but now that you point it out they sound like Yoda talk. And I agree that `History` is a better nomenclature so they don't get confused with our logging framework.